src/class-wpml-gutenberg-integration.php

```diff
@@ -139,6 +139,7 @@ private function update_block_translations( $blocks, $string_translations, $lang
 
 			$block = self::sanitize_block( $block );
 			$block = $this->strings_in_blocks->update( $block, $string_translations, $lang );
+			$block = apply_filters( 'wpml_block_additional_strings', $block, $string_translations, $lang );
```

Comment:
@kkarpieszuk, IMO this filter hook should be located inside `$this->strings_in_blocks->update()` and I would rename it `wpml_update_strings_in_block`.

The location of this hook will be consistent with the other one.

Reply:
done